fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -1265,6 +1269,12 @@ alterLoadStatement
         jobProperties?
     ;
 
+// ------------------------------------------- Compaction Statement ----------------------------------------------------------
+
+cancelCompactionStatement
+    : CANCEL COMPACTION (WHERE expression)?
```

Comment:
It seems that the where clause is required in the analysis phase, so how about defining the statement as `CANCEL COMPACTION WHERE expression`